build-tools/xaprepare/xaprepare/Steps/Step_GenerateFiles.cs

```diff
@@ -254,5 +264,14 @@ public GeneratedFile Get_Omnisharp_Json (Context context)
 				Path.Combine (BuildPaths.XamarinAndroidSourceRoot, OutputFileName)
 			);
 		}
+
+		public GeneratedFile Get_SourceLink_Json (Context context)
+		{
+			return new GeneratedSourceLinkJsonFile (
```

Comment:
I think the PR is missing the `GeneratedSourceLinkJsonFile` sources :(

Reply:
Doh!